task/06-conditions-n-loops-tasks.js

```diff
@@ -226,7 +257,8 @@ function reverseString(str) {
  *   34143 => 34143
  */
 function reverseInteger(num) {
-    throw new Error('Not implemented');
+    let reversed = num.toString().split('').reverse().join('');
+    return Math.sign(num) * parseInt(reversed); 
```

Comment:
The test can be passed even without Math.sign, but attention to the details is excellent! Thanks